### PR TITLE
Update navigation.yml

### PR DESCRIPTION
**Pull request by OpenGlobe**
Replaces the awardees navigation found here: 

https://www.nsf.gov/eng/iip/sbir/grantees.jsp

Fixes issue(s) # .

[CircleCI](https://circleci.com/gh/18F/nsf-sbir/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/nsf-sbir/awardees-side-nav/)

[Preview README for this branch](https://github.com/18F/nsf-sbir/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

/cc @relevant-people

### CODE REVIEW

**Comment by OpenGlobe** [comment]
I'm now pulled in Kate's edits...

File: pages/awardees/awardees.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: For Awardees
+permalink: /awardees/awardees/
```
Comment (gemfarmer) [inline pages/awardees/awardees.md]: Do we want this to just be `/awardees/`?

Comment (OpenGlobe) [inline pages/awardees/awardees.md]: Yeah, I think so - good catch.

Comment (OpenGlobe) [inline pages/awardees/awardees.md]: On second thought - what we are basing this off of is: 

https://www.nsf.gov/eng/iip/sbir/grantees.jsp

I think we want this page to render as the default page when going to the /awardees/ dir -

Comment (gemfarmer) [inline pages/awardees/awardees.md]: https://www.nsf.gov/eng/iip/sbir/grantees.jsp as the default page? Or what you currently have as `awardees.md`?

File: _data/navigation.yml [hunk 2]
```diff
@@ -28,6 +28,30 @@ fastlane:
     href: /fastlane/fastlane-2-3/
   - text: Proposal Submission
     href: /fastlane/fastlane-3/
+    
+awardees:
+  - text: General Grant Conditions
+    href: https://www.nsf.gov/awards/managing/special_conditions.jsp
```
Comment (line47) [inline _data/navigation.yml]: We need to add logic in here for using external links - right now these 2 links don't work

File: pages/awardees/awardees.md (hunk 1 above)
Comment (OpenGlobe) [inline pages/awardees/awardees.md]: Awardees.md (just modeled after the previous)

**Comment by line47** [comment]
This is ready for your 👁  @gemfarmer

**Pull request merged**